x-pack/legacy/plugins/apm/public/components/app/Settings/ListSettings.tsx

```diff
@@ -0,0 +1,290 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useState } from 'react';
+import moment from 'moment';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import {
+  EuiTitle,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButtonEmpty,
+  EuiPanel,
+  EuiBetaBadge,
+  EuiSpacer,
+  EuiCallOut,
+  EuiEmptyPrompt,
+  EuiButton,
+  EuiLink
+} from '@elastic/eui';
+import { loadCMList } from '../../../services/rest/apm/settings';
+import { useFetcher } from '../../../hooks/useFetcher';
+import { ITableColumn, ManagedTable } from '../../shared/ManagedTable';
+import { CMListAPIResponse } from '../../../../server/lib/settings/cm/list_configurations';
+import { AddSettingsFlyout } from './AddSettings/AddSettingFlyout';
+import { APMLink } from '../../shared/Links/APMLink';
+
+export type Config = CMListAPIResponse[0];
+
+export function ListSettings() {
+  const { data = [], refresh } = useFetcher(loadCMList, []);
+  const [selectedConfig, setSelectedConfig] = useState<Config | null>(null);
+  const [isFlyoutOpen, setIsFlyoutOpen] = useState(false);
+
+  const COLUMNS: Array<ITableColumn<Config>> = [
+    {
+      name: i18n.translate(
+        'xpack.apm.settings.cm.configTable.serviceNameColumnLabel',
+        {
+          defaultMessage: 'Service name'
```

Comment:
Any reason why sortable isn't true for Service name? I'm playing around with this PR and it seems weird that I can't sort by Service name but can for all other table headers.

Reply:
Yeah, that column should be sortable. Also, I think we should default to sort by service.

Reply:
Thanks for pointing that out! 😄